auxx/src/Command/TyProjection.hs

```diff
@@ -149,10 +149,10 @@ tyProposeUpdateSystem :: TyProjection ProposeUpdateSystem
 tyProposeUpdateSystem = TyProjection "ProposeUpdateSystem" (preview _ValueProposeUpdateSystem)
 
 tySystemTag :: TyProjection SystemTag
-tySystemTag = TyProjection "SystemTag" ((fmap . fmap) (SystemTag . fromString) (preview _ValueString))
+tySystemTag = TyProjection "SystemTag" ((fmap . fmap) (UnsafeSystemTag . fromString) (preview _ValueString))
 
 tyApplicationName :: TyProjection ApplicationName
-tyApplicationName = TyProjection "ApplicationName" ((fmap . fmap) (ApplicationName . fromString) (preview _ValueString))
+tyApplicationName = TyProjection "ApplicationName" ((fmap . fmap) (UnsafeApplicationName . fromString) (preview _ValueString))
```

Comment:
We should drop this `Unsafe*` naming convention. There's nothing unsafe about using a constructor. It doesn't risk any segmentation fault or undefined behaviour or breakage of referential transparency.

Reply:
Let others decide. I find it to be pretty neat. cc @gromakovsky @int-index

Reply:
It's certainly good naming convention for such Unchecked constructors. Maybe only we should use `Unchecked` or other prefix instead of `Unsafe`

Reply:
After recent changes I don't like `Unsafe` prefix. `Unchecked` is ok.